### PR TITLE
ERMS-2546, ERMS-2327, ERMS-2556,ERMS-2547 

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/services/de/laser/NormalizeService.groovy [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package de.laser
+
+import grails.transaction.Transactional
+
+@Transactional
+class NormalizeService {
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/NormalizeService.groovy:6]: Unnecessary; there is the EscapeService created for this purpose.

File: app/grails-app/services/de/laser/SurveyService.groovy [hunk 2]
```diff
@@ -28,6 +28,7 @@ class SurveyService {
     Locale locale
     EscapeService escapeService
     MailService mailService
+    NormalizeService normalizeService
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/SurveyService.groovy:31]: See two lines above. Use EscapeService instead.

File: app/grails-app/services/de/laser/SurveyUpdateService.groovy [hunk 3]
```diff
@@ -15,6 +15,7 @@ class SurveyUpdateService extends AbstractLockableService {
 
     def grailsApplication
     def mailService
+    NormalizeService normalizeService
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/SurveyUpdateService.groovy:18]: Use EscapeService

File: app/grails-app/taglib/de/laser/LaserInjectionTagLib.groovy [hunk 4]
```diff
@@ -21,5 +21,6 @@ class LaserInjectionTagLib {
         g.set( var:'taskService', bean:'taskService' )
         g.set( var:'GOKbService', bean:'GOKbService' )
         g.set( var:'yodaService', bean:'yodaService' )
+        g.set(var:'normalizeService', bean: 'normalizeService')
```
Comment (croqueGrec09) [inline app/grails-app/taglib/de/laser/LaserInjectionTagLib.groovy:24]: Use escapeService

File: app/grails-app/views/mailTemplates/text/dashboardDueDates.gsp [hunk 5]
```diff
@@ -8,8 +8,8 @@ ${message(code: 'profile.dashboardReminderEmailText2', locale: language, args:[o
 
 ----------------------------------------------------------------------------------------------------------------------------------
 <g:each in="${dueDates}" var="dashDueDate"><g:set var="obj" value="${genericOIDService.resolveOID(dashDueDate.dueDateObject.oid)}"/><g:if test="${obj}">
-    ${raw(language == RDStore.LANGUAGE_DE ? dashDueDate.dueDateObject.attribute_value_de :
-            dashDueDate.dueDateObject.attribute_value_en)}
+    ${raw(language == RDStore.LANGUAGE_DE ? normalizeService.replaceUmlaute(dashDueDate.dueDateObject.attribute_value_de) :
```
Comment (croqueGrec09) [inline app/grails-app/views/mailTemplates/text/dashboardDueDates.gsp:11]: use escapeService

File: app/grails-app/views/mailTemplates/text/notificationSurvey.gsp [hunk 6]
```diff
@@ -2,7 +2,7 @@
 ${message(code: 'email.text.title', locale: language)} ${userName},
 
 ${message(code: 'email.survey.text', locale: language)}
-${survey.name} (<g:formatDate format="${message(code:'default.date.format.notime', default:'yyyy-MM-dd')}" date="${survey.startDate}"/> - <g:formatDate format="${message(code:'default.date.format.notime', default:'yyyy-MM-dd')}" date="${survey.endDate}"/>)
+${normalizeService.replaceUmlaute(survey.name)} (<g:formatDate format="${message(code:'default.date.format.notime', default:'yyyy-MM-dd')}" date="${survey.startDate}"/> - <g:formatDate format="${message(code:'default.date.format.notime', default:'yyyy-MM-dd')}" date="${survey.endDate}"/>)
```
Comment (croqueGrec09) [inline app/grails-app/views/mailTemplates/text/notificationSurvey.gsp:5]: use EscapeService

File: app/test/unit/de/laser/NormalizeServiceSpec.groovy [hunk 7]
```diff
@@ -0,0 +1,20 @@
+package de.laser
```
Comment (croqueGrec09) [inline app/test/unit/de/laser/NormalizeServiceSpec.groovy:1]: unnecessary as service is not needed

File: app/grails-app/i18n/messages_de.properties [hunk 8]
```diff
@@ -1911,6 +1911,7 @@ subscription.details.manageEntitlementGroup.label = Titel-Gruppen verwalten
 subscription.details.issueEntitlementEnrichment.label = Titel-Anreicherung
 subscription.details.issueEntitlementEnrichment.uploadCoverageDates.label = Abdeckungsdaten hinzufügen
 subscription.details.issueEntitlementEnrichment.uploadPriceInfo.label = Preisangaben hinzufügen
+subscription.details.issueEntitlementEnrichment.enrichmentProcess = <b>Titel-Anreicherung erfolgreich:</b> Verarbeitet {1} von {0}. <b>Abdeckungsdaten hinzufügt:</b> {2}.  <b>Preisangaben hinzufügt:</b> {3}
```
Comment (Inselhopper) [inline app/grails-app/i18n/messages_de.properties:1914]: IST: <b>Preisangaben hinzufügt:</b>
SOLL: <b>Preisangaben hinzugefügt:</b>

**Pull request merged**